transformer/catalog-transformer-tika-input/src/main/java/ddf/catalog/transformer/input/tika/TikaInputTransformer.java

```diff
@@ -189,8 +197,44 @@ private Metacard createMetacard(InputStream input, Metadata metadata, String uri
         return metacard;
     }
 
-    private String toWkt(String lon, String lat) {
+	private String getCoordinates(String cornerCoordinates, String corner) {
```

Comment:
It looks like most of this implementation can be replaced with StringUtils.substrinbBetween(StringUtils.substringAfter(cornerCOordinates, corner),"(",")").trim().

https://commons.apache.org/proper/commons-lang/javadocs/api-2.6/org/apache/commons/lang/StringUtils.html#substringAfter(java.lang.String, java.lang.String)

https://commons.apache.org/proper/commons-lang/javadocs/api-2.6/org/apache/commons/lang/StringUtils.html#substringBetween(java.lang.String, java.lang.String, java.lang.String)